core/src/main/java/com/found404/core/models/CreditCardType.kt

```diff
@@ -1,6 +1,13 @@
 package com.found404.core.models
-
+enum class CardBrandType(val cardBrandId: Int, val brandName: String) {
+    DINERS(1, "Diners"),
+    MASTERCARD(2, "MasterCard"),
+    VISA(3, "VISA"),
+    DISCOVER(4, "Discover"),
+    MAESTRO(5, "Maestro");
+}
```

Comment:
This should be removed and cardbrands from the backend need to be used. It is enough to define CreditCardType here.